recipes/cmappy/meta.yaml

```diff
@@ -4,7 +4,7 @@ package:
 
 source:
   fn: cmapPy-2.1.0.tar.gz
-  url: https://pypi.python.org/packages/f1/08/b803a94071566ff86dfd7debd86b96f85f741ebf4b7ffaf601d901005b5e/cmapPy-2.1.0.tar.gz#
+  url: https://pypi.python.org/packages/f1/08/b803a94071566ff86dfd7debd86b96f85f741ebf4b7ffaf601d901005b5e/cmapPy-2.1.0.tar.gz
```

Comment:
You want:
https://pypi.python.org/packages/cb/8a/e0349bcfee1243f20d1bd2f52a37b026733002f9cceb4c4c5fa5e253c7af/cmapPy-2.1.0.tar.gz